src/frontend/web_application/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# This file creates a container that runs a {package} caliopen frontend
+# Important:
+# Author: Caliopen
+# Date: 2016-01-05
+
+FROM node:6
+MAINTAINER Caliopen
+
+ADD . /srv/caliopen/frontend/
+WORKDIR /srv/caliopen/frontend/
+RUN yarn install
+RUN yarn run release
```

Comment:
I would like to go for hot-reload in an other PR, it has more side effects that just changing an env or a command.

I know this PR is stable and does what is important right now: build the current code and start in an optimized  way.

I have no time to polish this PR and I'm not sure it is efficient right now:
* if you are a python/golang hacker: just change the code rebuild and go: `docker-compose build frontend && docker-compose stop frontend  && docker-compose up -d frontend`
* if you are a frontend dev: install, node, yarn, launch the app  `yarn start` then dev (with hot-reload)

WDYT ?